examples/package.json

```diff
@@ -5,13 +5,15 @@
   "main": "example.js",
   "scripts": {
     "start": "node example.js",
-    "test": "jest --config=jest.config.ts"
+    "test": "node -r ts-node/register node_modules/.bin/jest --config=jest.config.ts"
```

Comment:
This was the main reason I didn't do this in the first place (actually I had TS types in the first place and commented them out), but seems worth it now that we're importing `build-roc` directly in other projects!